src/Agent.Sdk/Util/VssUtil.cs

```diff
@@ -21,6 +21,30 @@ public static class VssUtil
     {
         private static UtilKnobValueContext _knobContext = UtilKnobValueContext.Instance();
 
+        public static bool IsCustomServerCertificateValidationSupported
```

Comment:
I think this should be a cached property instead, so that we only calculate it's value once.